docs/features/software-templates/writing-custom-actions.md

```diff
@@ -187,7 +182,9 @@ export const examples: TemplateExample[] = [
 
 Add the example to the `createTemplateAction` under the object property `examples`:
 
-`return createTemplateAction<{ contents: string; filename: string }>({id: 'acme:file:create', description: 'Create an Acme file', examples, ...};`
+```ts
+return createTemplateAction<{ contents: string; filename: string }>({id: 'acme:file:create', description: 'Create an Acme file', examples, ...});
```

Comment:
Is this correct? Or maybe there is a better example we could add here?

Reply:
This way is deprecated now in favor of the new format:

https://github.com/backstage/backstage/pull/28798/files#diff-b87c64cc6e9b4dd65906e9897cf4b357d863163f6d06ed177a7f7e81dbebfb1dR54-R72

Reply:
should we update this to use the new format?

Reply:
fancy. no worries! :pray:

Reply:
Hey @benjdlambert 👋 

Can you just double check https://github.com/backstage/backstage/pull/29612/commits/247bd4fe219eefec321c336e9920c398565748e5 to make sure its 👌